aws/database/README.md

```diff
@@ -0,0 +1,16 @@
+# AWS Database module
+Use this module for deploying RDS instances in AWS
+
+## Usage
+Example on how to use module
+
+```
+module "database" {
+	# TODO: Fill me
```

Comment:
@The-smooth-operator Can you replace this TODO before merging?

Reply:
oh sure I forgot to add it. Thanks for catching this